finalPrep/README.md

```diff
@@ -23,52 +23,52 @@ Some things to consider as you refactor include...
+- [X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- Week 1 Screenshot ![Screen Shot 2022-02-10 at 4 05 21 PM](https://user-images.githubusercontent.com/98493391/153512254-56d67246-1a96-4168-a25d-c92cc417f7f1.png)
+- Week 2 Screenshot![Screen Shot 2022-02-10 at 4 05 32 PM](https://user-images.githubusercontent.com/98493391/153512447-478945a6-7c69-4bda-932b-da0a1dd4deb0.png)
+- Week 3 Screenshot ![Screen Shot 2022-02-10 at 4 05 42 PM](https://user-images.githubusercontent.com/98493391/153512499-64748e92-eae8-4b17-ab33-75c4c76106c7.png)
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [DTR Prep](https://gist.github.com/catlynbowles/4a0781d0659bef275894a3d4a5c031f5)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
-
+- [X] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [X] Add a link to your gist here: [Pre Teaching Gist](https://gist.github.com/catlynbowles/7cd5524c60237d511b81623ca3da4890) 
+  
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: [DTR Guiding Gist](https://gist.github.com/catlynbowles/5bc96c3fe5df223446e1992c85506c71)
```

Comment:
I think your group and mentor DTRs show you are self aware and ready to work really effectively in collaborative spaces!